### PR TITLE
css fixes for 2023.1

### PR DESCRIPTION
**Pull request by sadiqkhoja**
- Add page icon with the form name on form overview page
- Minor css fixes on Dataset overview page

### CODE REVIEW

File: src/assets/scss/app.scss [hunk 1]
```diff
@@ -814,3 +814,9 @@ becomes more complicated.
     }
   }
 }
+
+.title-icon{
```
Comment (matthew-white) [inline src/assets/scss/app.scss:818]: There's a section above for icons, would you mind removing this there?

File: src/components/form/new.vue [hunk 2]
```diff
@@ -248,13 +248,13 @@ export default {
         });
     },
     removeLearnMore(value) {
-      return value.replace(/Learn more: (https?:\/\/xlsform\.org\/[^.]*)\.?/, '').trim();
+      return value.replace(/Learn more: (https?:\/\/xlsform\.org[^.]*)\.?/, '').trim();
```
Comment (matthew-white) [inline src/components/form/new.vue:251]: I think it'd be nice to verify the character after .org. (Wouldn't want to match on xlsform.org.uk. 🤣) How about `\.org[/#]`?

File: src/components/dataset/overview/dataset-properties.vue [hunk 3]
```diff
@@ -70,17 +70,23 @@ export default {
...
-  }
+  &.table{
+    table-layout: fixed;
 
-  &.table > tbody > tr:first-child > td {
-    border-top: none;
-  }
+    td {
+      // @include text-overflow-ellipsis;
+      word-wrap: break-word;
```
Comment (matthew-white) [inline src/components/dataset/overview/dataset-properties.vue:78]: I think this is deprecated, how about `overflow-wrap: break-word;` ?

**Review by matthew-white — APPROVED** [review]

File: src/components/form/new.vue (hunk 2 above)
Comment (sadiqkhoja) [reply src/components/form/new.vue:251]: then it won't match with just `https://xlsform.org`

**Pull request merged**